lib/coverband/utils/gem_list.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+####
+# An array of  FileLists instances with helpers to roll up the stats
+# methods for calculating coverage across them etc.
+####
+module Coverband
+  module Utils
+    class GemList < FileList
+      # Returns the count of lines that have coverage
+      def covered_lines
```

Comment:
Using `to_a` on these methods instead of the empty check might work? 
```ruby
def covered_lines
  to_a.map(&:covered_lines).sum
end
```
Looks like calling `to_a` on an exiting array, just returns the same instance:
```
irb(main):012:0> a = [:test]
=> [:test]
irb(main):013:0> a.object_id == a.to_a.object_id
=> true
irb(main):014:0>
```
And `nil.to_a` returns `[]`.